misc/config_tools/data/whl-ipc-i5/sdc.xml

```diff
@@ -19,7 +19,6 @@
         <CDP_ENABLED>n</CDP_ENABLED>
```

Comment:
@liuhang-bit What do you think about removing this file, since we no longer support sdc scenario?

Reply:
yes we could remove sdc scenario, since we do not support anymore, but I do not suggest to do that in this PR.

Reply:
Agree to remove if no document link to this file.

Reply:
@liuhang-bit Sounds good. Please remove in a future pull request.